config.example.js

```diff
@@ -57,7 +57,12 @@ module.exports = {
 
    // List of repositories for pulldasher to watch.
    repos: [
-     "owner/name"
+      {
+         full_name: "owner/name",
```

Comment:
I don't really know how github describes the full name of a repo, so maybe this isn't helpful, but would it be better to store the org name and repo name as their own fields in the object and then have `getRepoFullNames` concat those individual values into the canonical `org/repo` string?

I can't think of a use case now, but if we ever wanted to get an org name out of a repo object, I think it'd be easier as a field access than having to parse out the name from `full_name`.

Reply:
I agree, it will probably come into use. The github object for it holds the full_name and the repo, owner all in separate variables.

I think this is totally reasonable to add when it is desired and could be simply implemented as parsing the full_name from the config.